Cod3rsGrowth.Web/wwwroot/repositorios/Api.js

```diff
@@ -0,0 +1,108 @@
+sap.ui.define(
+  ["sap/ui/core/mvc/Controller", "sap/ui/core/format/DateFormat"],
+  function (Controller, DateFormat) {
+    "use strict";
+
+    let oResourceBundle;
+    const urlApi = "http://localhost:5285/pecas";
+
+    const httpStatusOk = 200;
+    const httpStatusCreated = 201;
+    const httpStatusNoContent = 204;
+
+    return Controller.extend("sap.ui.cod3rsgrowth.repositorios.Api", {
+      onInit: function () {
+        oResourceBundle = this.getOwnerComponent()
+          .getModel("i18n")
+          .getResourceBundle();
+      },
+
+      carregarPecas: async function () {
+        try {
```

Comment:
Pode retirar os try, catch do repositorio, todos vão ficar onde você for chamar as requisições